ci-operator/config/openshift/microshift/openshift-microshift-release-4.13.yaml

```diff
@@ -115,76 +115,95 @@ tests:
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-openshift-conformance-reduced
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-openshift-conformance-reduced-aws
 - as: e2e-reboot
   cluster: build02
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
-    workflow: openshift-microshift-e2e-reboot-tests
+      EC2_AMI: ami-07f5ac8ddbcfbb96e
+    workflow: openshift-microshift-e2e-reboot-tests-aws
 - as: e2e-reboot-arm
   cluster: build02
   optional: true
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-reboot-tests
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-reboot-tests-aws
 - as: e2e-greenboot
   cluster: build02
   optional: true
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|scripts/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-v20230203
-    workflow: openshift-microshift-e2e-greenboot
+      EC2_AMI: ami-07f5ac8ddbcfbb96e
+    workflow: openshift-microshift-e2e-greenboot-aws
 - as: e2e-greenboot-arm
   cluster: build02
   optional: true
   run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|scripts/.*|vendor/.*|go\.sum|go\.mod|Makefile)
   steps:
-    cluster_profile: gcp
+    cluster_profile: aws
     env:
-      GOOGLE_COMPUTE_IMAGE_NAME: rhel-9-arm64-v20230306
-      GOOGLE_COMPUTE_MACHINE_TYPE: t2a-standard-4
-    workflow: openshift-microshift-e2e-greenboot
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-greenboot-aws
+- as: microshift-e2e
+  cluster: build02
+  optional: true
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  steps:
+    cluster_profile: aws
+    env:
+      EC2_AMI: ami-07f5ac8ddbcfbb96e
+    workflow: openshift-microshift-e2e-microshift-tests-aws
+- as: microshift-e2e-arm
+  cluster: build02
+  optional: true
+  run_if_changed: ^(assets/.*|cmd/.*|etcd/.*|pkg/.*|packaging/.*|vendor/.*|go\.sum|go\.mod|Makefile|.*\.sh)
+  steps:
+    cluster_profile: aws
+    env:
+      EC2_AMI: ami-0c231be9fb222349e
+      EC2_INSTANCE_TYPE: a1.2xlarge
+    workflow: openshift-microshift-e2e-microshift-tests-aws
```

Comment:
These should only be part of 4.14+ versions, as they are not defined in release-4.13 or lower branches in microshift.